src/index.js

```diff
@@ -58,15 +58,15 @@ function requireAllModels(modelsDir) {
         recursive: true,
       });
     } catch (error) {
-      logger.error(`Cannot read a file for the following reason: ${error.message}`, error);
```

Comment:
I think the `error` printed after the error message is supposed to give some context (stack trace,...).
So I am wondering if we really want to do that?